src/chrome/chromeDebugAdapter.ts

```diff
@@ -152,11 +153,15 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
 
     private _batchTelemetryReporter: BatchTelemetryReporter;
 
-    public constructor({ chromeConnection, lineColTransformer, sourceMapTransformer, pathTransformer, targetFilter, enableSourceMapCaching }: IChromeDebugAdapterOpts, session: ChromeDebugSession) {
+    public readonly Events: StepProgressEventsEmitter;
```

Comment:
Why capital E?

Reply:
I think I was thinking in C#... I'll change it to lowercase...